editor.planx.uk/src/@planx/components/shared/Preview/SummaryList.tsx

```diff
@@ -75,6 +77,76 @@ const components: {
   [TYPES.TextInput]: TextInput,
 };
 
+interface SummaryListsBySectionsProps {
+  breadcrumbs: Store.breadcrumbs;
+  flow: Store.flow;
+  passport: Store.passport;
+  changeAnswer: (id: Store.nodeId) => void;
+  showChangeButton: boolean;
+  sectionComponent: React.ElementType<any> | undefined;
+}
```

Comment:
i imagine this might need a future prop like `showSectionStatus` or something similar when we get to the Save&Return reconciliation tickets which would additionally calculate & display the status tag next to the section headers, but I think plain subheaders satisfy the design for now!